lib/css/exports/_stacks-constants-colors.less

```diff
@@ -250,20 +256,20 @@ body {
     --red-800: @red-800;
     --red-900: @red-900;
 
-    // Badges
-    --badge-gold:    @badge-gold;
-    --badge-silver:  @badge-silver;
-    --badge-bronze:  @badge-bronze;
-
-    --gold-dot:      @gold-dot;
-    --gold-bg:       @gold-bg;
-    --gold-border:   @gold-border;
-    --silver-dot:    @silver-dot;
-    --silver-bg:     @silver-bg;
-    --silver-border: @silver-border;
-    --bronze-dot:    @bronze-dot;
-    --bronze-bg:     @bronze-bg;
-    --bronze-border: @bronze-border;
+    // Gold
+    --gold: @gold;
```

Comment:
Potential breaking change, but I doubt anyone is using our css variables in any capacity, much less these badge variables.